snuba/utils/metrics/backends/testing.py

```diff
@@ -93,3 +93,20 @@ def timing(
             assert isinstance(value, (int, float))
             if tags is not None:
                 self.__validate_tags(tags)
+
+    def events(
+        self,
+        title: str,
+        text: str,
+        alert_type: str,
+        priority: str,
+        tags: Optional[Tags] = None,
+    ) -> None:
+        record_metric_call("events", title, 1, tags)
```

Comment:
You should probably define something like `record_event_call` and make it generic for events. Fitting events into metric does not seem right